### PR TITLE
fix(faq): added instance pricing MTA-2007

### PR DESCRIPTION
**Pull request by nedaehm**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](../docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: faq/instances.mdx [hunk 1]
```diff
@@ -20,6 +20,57 @@ You must [create at least one snapshot](/compute/instances/how-to/create-a-snaps
...
+### Storage Pricing
+
+| Product       | Unit        | All regions* | PAR3        |
+|---------------|-------------|--------------|-------------|
+| Local Storage | Gb/hour     | €0.0000400   | €0.0000400  |
+| Local Snapshot| Gb/hour     | €0.0000400   | €0.0000400  |
+| Block Storage | Gb/hour     | €0.0001070   | €0.0003210  |
+| Block Snapshot| Gb/hour     | €0.0000268   | €0.0000804  |
+
+*All regions include: PAR1, PAR2, AMS1, AMS2 and WAW1. PAR3 is not included.
```
Comment (RoRoJ) [inline faq/instances.mdx:68]: sorry, I think if the one above is changed this one should be too!

**Review by RoRoJ — APPROVED** [review]

File: faq/instances.mdx [hunk 2]
```diff
@@ -8,6 +8,57 @@ content:
...
+| General Purpose | GP1-XL        | hour    | €1.4920      | €2.2380  |
+| Enterprise      | ENT1-S        | hour    | €0.2800      | €0.4200  |
+| Enterprise      | ENT1-M        | hour    | €0.5600      | €0.8400  |
+| Enterprise      | ENT1-L        | hour    | €1.1200      | €1.6800  |
+| Enterprise      | ENT1-XL       | hour    | €2.2400      | €3.3600  |
+| Enterprise      | ENT1-2XL      | hour    | €3.3600      | €5.0400  |
+| GPU             | Render-S      | hour    | €1.0620      |          |
+| GPU             | GPU-3070-S    | hour    | €0.8500      |          |
+
+*All regions includes PAR1, PAR2, AMS1, AMS2 and WAW1. PAR3 prices are shown separately.
```
Comment (ofranc) [inline faq/instances.mdx:45]:
```suggestion
*All regions include `PAR1`, `PAR2`, `AMS1`, `AMS2`, and `WAW1`. `PAR3` prices are shown separately. 
```

File: faq/instances.mdx [hunk 3]
```diff
@@ -8,6 +8,57 @@ content:
...
+### Storage Pricing
+
+| Product       | Unit        | All regions* | PAR3        |
+|---------------|-------------|--------------|-------------|
+| Local Storage | Gb/hour     | €0.0000400   | €0.0000400  |
+| Local Snapshot| Gb/hour     | €0.0000400   | €0.0000400  |
+| Block Storage | Gb/hour     | €0.0001070   | €0.0003210  |
+| Block Snapshot| Gb/hour     | €0.0000268   | €0.0000804  |
+
+*All regions include: PAR1, PAR2, AMS1, AMS2 and WAW1. PAR3 prices are shown separately.
```
Comment (ofranc) [inline faq/instances.mdx:56]:
```suggestion
*All regions include `PAR1`, `PAR2`, `AMS1`, `AMS2` and `WAW1`. `PAR3` prices are shown separately.
```

**Review by ofranc — APPROVED** [review]

File: faq/instances.mdx [hunk 4]
```diff
@@ -8,6 +8,57 @@ content:
...
+
+Compute Instances are billed on a pay-as-you-go basis. The price of your Instance is made up of three parts:
+
+* the price of the Instance itself
+* the price of the storage (local or block). You can adjust storage quantity when you create the Instance
+* the price of the flexible IP - you can change or delete associated flexible IP afterward
+
+### Instance Pricing as of July 1st 2022
+
+<Message type="important">
```
Comment (ldecarvalho-doc) [inline faq/instances.mdx:21]:
```suggestion
<Message type="important">

```

File: faq/instances.mdx [hunk 5]
```diff
@@ -8,6 +8,57 @@ content:
...
+Prices indicated below do not include storage or IP. Bear in mind that the following prices will be effective starting July 1st 2022.
```
Comment (ldecarvalho-doc) [inline faq/instances.mdx:22]:
```suggestion
Prices indicated below do not include storage or IP. Bear in mind that the following prices will be effective starting July 1st 2022.

```

**Pull request merged**